vantage6/node/__init__.py

```diff
@@ -37,6 +38,20 @@
 from vantage6.node.util import logger_name
 
 
+# TODO BvB 2021-08-30: I'm not happy having this class here, but not sure where
```

Comment:
I think I would move this to the `vantage6-common` repository. We should also do the same for the server instance.